rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -364,7 +136,7 @@ def calculate_claimed_airdrops(
         amount = amount_with_tolerance.value
         half_range = amount_with_tolerance.tolerance / 2
         query_parts.append('events.location_label=? AND events.asset=? AND CAST(events.amount AS REAL) BETWEEN ? AND ?')  # noqa: E501
-        bindings.extend([airdrop_info[0], airdrop_info[1].serialize(), str(amount - half_range), str(amount + half_range)])  # noqa: E501
+        bindings.extend([airdrop_info[0], airdrop_info[1], str(amount - half_range), str(amount + half_range)])  # noqa: E501
```

Comment:
can you modify the return of this function? if you notice it is iterating twice: once in the fetchall and the other in the comprehension list. I believe taht you can do `return cursor.execute(...).fetchall()`